console/console-init/ui/src/modules/address-detail/containers/AddressLinks/AddressLinksPage.tsx

```diff
@@ -118,7 +78,12 @@ const AddressLinksPage: React.FunctionComponent<IAddressLinksListPageProps> = ({
             />
           </GridItem>
           <GridItem span={5}>
-            {addresLinksTotal > 0 && renderPagination(page, perPage)}
+            <TablePagination
```

Comment:
The same component is being used with different values, better way would be to create a method like renderPagination like we had earlier and pass different arguments to it in both places.